relay-general/src/store/normalize/contexts.rs

```diff
@@ -177,7 +178,42 @@ fn normalize_os_context(os: &mut OsContext) {
     }
 }
 
+fn parse_raw_response_data(response: &mut ResponseContext) -> Option<(&'static str, Value)> {
+    let raw = response.data.as_str()?;
+
+    // TODO: Try to decode base64 first
+
+    if let Ok(value) = serde_json::from_str(raw) {
+        Some(("application/json", value))
+    } else {
+        url_encoding::encoded_from_str(raw)
+            .map(|value| ("application/x-www-form-urlencoded", value))
```

Comment:
This content type makes sense as a fallback for requests, but it seems wrong for responses. Do HTTP responses actually contain urlencoded form data?

Reply:
Based on a quick search, https://stackoverflow.com/questions/4007969/application-x-www-form-urlencoded-or-multipart-form-data but the response was still JSON.
My question is then, what would be the fallback if any? the original `response.data`?

Reply:
> My question is then, what would be the fallback if any? the original response.data?

Yes, I would just return `None` here if JSON parsing fails, and leave the `data` field untouched (same as what would happen now if parsing of form data fails).